src/docs/stan-reference/examples.tex

```diff
@@ -7000,12 +7000,10 @@ \section{Simulating from a Gaussian Process}
   real x[N];
 }
 transformed data {
-  vector[N] mu;
-  cov_matrix[N] K;
-  mu = rep_vector(0, N);
-  K = cov_exp_quad(x, 1, 1);
-  for (i in 1:N)
-    K[i, i] = K[i, i] + 0.1;
+  matrix[N, N] K = cov_exp_quad(x, 1.0, 1.0);
```

Comment:
And that's a very good point about efficiency.  I'll drop a note to that effect into the efficiency chapter.